packages/nouns-contracts/test/descriptor.test.ts

```diff
@@ -1,105 +0,0 @@
-import chai from 'chai';
-import { solidity } from 'ethereum-waffle';
-import { NounsDescriptor } from '../typechain';
-import ImageData from '../files/image-data-v1.json';
-import { LongestPart } from './types';
-import { deployNounsDescriptor, populateDescriptor } from './utils';
-import { ethers } from 'hardhat';
-import { appendFileSync } from 'fs';
-
-chai.use(solidity);
-const { expect } = chai;
-
-describe('NounsDescriptor', () => {
-  let nounsDescriptor: NounsDescriptor;
-  let snapshotId: number;
-
-  const part: LongestPart = {
-    length: 0,
-    index: 0,
-  };
-  const longest: Record<string, LongestPart> = {
-    bodies: part,
-    accessories: part,
-    heads: part,
-    glasses: part,
-  };
-
-  before(async () => {
-    nounsDescriptor = await deployNounsDescriptor();
-
-    for (const [l, layer] of Object.entries(ImageData.images)) {
-      for (const [i, item] of layer.entries()) {
-        if (item.data.length > longest[l].length) {
-          longest[l] = {
-            length: item.data.length,
-            index: i,
-          };
-        }
-      }
-    }
-
-    await populateDescriptor(nounsDescriptor);
-  });
-
-  beforeEach(async () => {
-    snapshotId = await ethers.provider.send('evm_snapshot', []);
-  });
-
-  afterEach(async () => {
-    await ethers.provider.send('evm_revert', [snapshotId]);
-  });
-
-  it('should generate valid token uri metadata when data uris are disabled', async () => {
```

Comment:
@eladmallel I didn't find an equivalent test in foundry

Reply:
decided it's not important to keep Descriptor tests when we are live with V2 (this is testing V1).